# goal
string arm_id
geometry_msgs/Pose pos
---
# result
bool success
string message
---
# feedback
float64 completion_percentage
geometry_msgs/Pose current_pose
